components/StandardListItem/StandardListItemStyles.scss

```diff
@@ -0,0 +1,58 @@
+.StandardListItem {
+
+  display: -webkit-flex;
```

Comment:
Do we really need webkit specific css? We are good if it works, without specific css, with last 2 versions of the webkit browsers. If you agree, please remove webkit specific rules from other places as well.